src/pages/dashboard/Dashboard.jsx

```diff
@@ -1,22 +1,49 @@
 import React, { useEffect, useState } from 'react';
+import { instanceOf } from 'prop-types';
 import { Flex, Heading, Stack, Text } from '@chakra-ui/react';
 import Logout from '../../components/Logout/Logout';
 import { getUserFromDB } from '../../utils/AuthUtils';
+import DriverNavbar from '../../components/Navbar/DriverNavbar';
+import AdminNavbar from '../../components/Navbar/AdminNavbar';
+import { withCookies, Cookies, cookieKeys } from '../../utils/CookieUtils';
+import AUTH_ROLES from '../../utils/AuthConfig';
 
-const Dashboard = () => {
+const { SUPERADMIN_ROLE, ADMIN_ROLE, DRIVER_ROLE } = AUTH_ROLES.AUTH_ROLES;
+
+const Dashboard = ({ cookies }) => {
   const [user, setUser] = useState({});
+  const [isSuperAdmin, setIsSuperAdmin] = useState(false);
+  const [isAdmin, setIsAdmin] = useState(false);
+  const [isDriver, setIsDriver] = useState(false);
```

Comment:
it's a bit verbose to add 3 state variables, you can do this in one by saving the role itself, then comparing it later on when conditionally rendering